__tests__/app.test.js

```diff
@@ -42,3 +42,21 @@ describe('/api/categories', () => {
         })
     });
 });
+
+describe("GET /api", () => {
+    test("status:200 responds with an object containing all current endpoints", () => {
+      return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body).toBe('object');
+        const expectedObject = {
+            "GET /api": expect.any(Object),
```

Comment:
This test will not be robust enough when you later expand the endpoints. Think about how you can compare the response to the endpoints.json file